Topics/python-fundamentals.md

```diff
@@ -0,0 +1,236 @@
+
+#### Stack Overflow
+
+You _should_ fall in love with [**StackOverflow**](https://stackoverflow.com/), a question-answer website about programming in general.
+Read [How can I use Stack Overflow effectively as a beginner?](https://www.quora.com/How-can-I-use-Stack-Overflow-effectively-as-a-beginner) and be ready to become grateful to awesome people around the world who answer questions daily on the website.
+
+### Develop Small Projects
+
+First of all: [**Read this post from r/LearnProgramming**](https://www.reddit.com/r/learnprogramming/comments/5zc24o/the_best_way_to_learn_how_to_code/), and stare it in your mind!
+
+**The biggest lesson in the world of programming is the following: there is no better lesson than getting your hands dirty!**
+
+Of course, to start with it makes sense to follow tutorials and guides (better still a structured book), but you need to experiment, make mistakes, and re-iterate this process to **can really improve** in your coding skills.
+
+Once you are done with the book "Automate the Boring Stuff", jump in a small project and develop it by yourself! 
+
+The best case is that you find a topic or a task in which you have an interest, and tackle it! 
+
+If you lack imagination don't worry, here you have a list of + 1000 project ideas you can develop in Python :)
+
+- [1000+ Beginner Programming Projects](https://www.reddit.com/r/learnprogramming/comments/2a9ygh/1000_beginner_programming_projects_xpost/)
+
+Having a project keeps you motivated, don't underestimate it!
+
+### Learn Git and GitHub
+
+Git is a versioning system that allows you to always have under control every change in your code, be able to go back, and be sure that your code will never be lost!
+
+Git is defined as **Distributed Version Control System**: What does it mean?
+
+From [this article](https://www.freecodecamp.org/news/what-is-git-and-how-to-use-it-c341b049ae61/):
+
+- Control System: This basically means that Git is a content tracker. So Git can be used to store content — it is mostly used to store code due to the other features it provides.
+
+- Version Control System: The code which is stored in Git keeps changing as more code is added. Also, many developers can add code in parallel. So Version Control System helps in handling this by maintaining a history of what changes have happened. Also, Git provides features like branches and merges, which I will be covering later.
+
+- Distributed Version Control System: Git has a remote repository that is stored in a server and a local repository that is stored in the computer of each developer. This means that the code is not just stored in a central server, but the full copy of the code is present in all the developers’ computers. Git is a Distributed Version Control System since the code is present in every developer’s computer. I will explain the concept of remote and local repositories later in this article.
```

Comment:
> ...repositories later in this article.

You said you will explain later, where did you explain it eventually? I couldn't find it.

Reply:
tru